### DISCUSSION TITLE
Distance between two events on same day

### DISCUSSION THREAD

**Discussion by sqlworldwide** [dataplat/DataSaturdays#51] [Ideas]
I know PASS had limitations of 600 miles distance between two events on the same day. Members had a difference of opinion on that.  Should we consider something similar?

[a comment by an automated account is not shown]

**Comment by rabryst** [comment]
It was unenforceable in Canada, and frankly I don't see the point of an arbitrary distance limit. For the amount of effort it goes to putting together a free event, it would make sense for people in areas nearby to work together. If there's something they were trying to avoid, let's think of different ways to avoid them.

[a comment by an automated account is not shown]

**Comment by ghotz** [comment]
I would be against such a rule for a couple of reasons. For example the distance between Milano and Zürich is roughly 300km (186mi), a bit less than going to Pordenone, yet the local communities speak different languages and many prefer to attend sessions (and network) in local language. So, while two events on the same day in Milan and Zürich would not compete, the same would not be true with Milan and Pordenone. Also, what I've experienced in the last few years speaking at Azure Bootcamps, running the same day in different major Italian cities, is more people attending across them (which just makes sense since they don't have to travel as much). It is more a burden for speakers that may need to choose which one to attend but that's it.

[a comment by an automated account is not shown]

**Comment by suxstellino** [comment]
What about distance by Country?

[a comment by an automated account is not shown]

**Comment by amtwo** [comment]
I think as an initial "rule" we can simply make sure that events on the same day know about each other & the organizers talk to each other.

I agree most attendees are unlikely to drive much more than an hour or two to go to a Saturday event.

One of the concerns I've heard in the past isn't about _attendees_ but rather about speakers. There was a complaint a few years ago that one of the bigger, more established events attracted all the good speakers and the smaller event on the same day struggled to get speakers.

However--speakers sometimes fly great distances to speak at events, particularly bigger events. If organizers are able to talk it out, they should be able to help each other, rather than fighting with each other.

If "make the organizers talk it out" proves to be ineffective, then I'd vote for revisiting this based on how things go.

[a comment by an automated account is not shown]

**Comment by sqlworldwide** [comment]
Agree with what @amtwo  said. Another concern is the sponsors. In the USA there are some sponsors who have national/global presence that goes to most events. Then you have a few local sponsors. If we have two events in close proximity as we will struggle with sponsors and what Andy said about speakers.

[a comment by an automated account is not shown]

**Comment by eugman** [comment]
My biggest concern would be sponsors for sure. I think that's the most inelastic of the three groups (Sponsors, speakers, attendees). 

Speakers is a valid concern, but I also feel like part of the goal of these events is to help encourage and grow local speakers. As for attendees, I agree this is likely a non-concern. My understanding of SQL Saturday Baltimore and DC is that they had very little attendee overlap, despite being very close in proximity.

[a comment by an automated account is not shown]

**Comment by way0utwest** [comment]
I have always disliked this rule, simple because it limits accessibility and the ability of organizers to reach the community. I believe this came about, as mentioned above, because of organizers not wanting to compete for speakers and sponsors being able to support multiple events.

My view has been that all of us our adults, and we can work out differences, or we will find that two events in the same city aren't sustainable for these, or other, reasons.

My guidance would be that organizers within a geography, say 100km, coordinate with each other.

I also hope this is a point of concern in 2021 and we have live events.

[a comment by an automated account is not shown]

**Comment by SQLDBAWithABeard** [comment]
I agree with @amtwo  and @rabryst on this issue. The distance between London and Paris is a touch over 200 miles for example but they would not be competing for sponsors or attendees I would surmise.

Communication between the events will be important - Maybe a calendar would be useful for organisers to be able to organise

[a comment by an automated account is not shown]

**Comment by amtwo** [comment]
Brainstorm🧠🌩:  What if there was a mechanism so that when an event date is entered, it automagically says "Here's the list of other events +/- a day of your event! Y'all should talk!" ... it should probably be also enable some sort of identical ping to the existing event organizers. That should make it easy for events held on the same day or same weekend to take note and decide if there's a problem. Paris & London organizers might decide to ignore the message, while Cincinnati & Cleveland might decide they need to talk & work together.

Event Date would need to be captured as some sort of structured/queryable metadata to enable that. 
The alert mechanism could be as simple as a @ mention in GitHub.

[a comment by an automated account is not shown]

**Comment by way0utwest** [comment]
I like that idea. Sharing information and allowing decentralized resolution are good qualities to embrace and support.

[a comment by an automated account is not shown]

**Comment by emdempster** [comment]
I like that idea from @amtwo to help event organizers know what's happening around them.  

This is a fringe case that may or may not need consideration, but in more remote areas, like the Great Plains and Mountain West in the US, limiting events on the same weekend, due to distance, may prevent some people from being able to attend.  As an example, let's say Denver CO and Casper WY want to run on the same weekend or weekends close to each other.  If Casper's event was rejected in favor of Denver, those who live in Rapid City, SD may be left out, because it's a longer drive to Denver than to Casper.

[a comment by an automated account is not shown]

**Comment by gavincampbell** [comment]
Regarding competition for sponsors and attendees, I don't think it matters that much whether the events are on the same day or not - whilst separating by date might make it _possible_ for attendees to attend a different event every Saturday for a month, in practice I think most people have a "time budget" for this kind of thing, and sponsors will have budgets that are quarterly or annual.
